tests/libjlm/tooling/TestJlcCommandLineParser.cpp

```diff
@@ -6,6 +6,7 @@
 #include "test-registry.hpp"
 
 #include <jlm/tooling/CommandLine.hpp>
+#include <jlm/tooling/CommandGraphGenerator.hpp>
```

Comment:
Is this include needed here? You are not doing command graph generation in this file.

Reply:
fixed